### PR TITLE
Add motor speed and direction control.

### PR DESCRIPTION
**Pull request by Meli-0xFF**
Signed-off-by: Martin Melichercik martin.melichercik@gmail.com

### CODE REVIEW

**Comment by Meli-0xFF** [comment]
done

File: programs/driver_board/main.c [hunk 1]
```diff
@@ -7,11 +7,17 @@
...
+#define MOTOR_PWM_CHANNEL 3
+
 #define UNUSED(arg) (void)(arg)
 
 thread_t *commanderp;
 
+int16_t motor_speed = 0;
+
 void execute_master_command(uint16_t command_id, uint8_t *buff) {
+    int16_t motor_speed;
```
Comment (pepol) [inline programs/driver_board/main.c]: Note that this local variable shadows the global one, making any changes to `motor_speed` within this function pointless if we want to work with them anywhere else (for example in `MotorThread`).

File: programs/driver_board/main.c [hunk 2]
```diff
@@ -19,7 +25,11 @@ void execute_master_command(uint16_t command_id, uint8_t *buff) {
 
         case 0xBB:
             // Motor movement command
-
+            if(buff[0] == 0) {
+                motor_speed = -1 * (buff[1] * 1000 + buff[2] * 100 + buff[3] * 10 + buff[4]);
+            } else {
+                motor_speed = (buff[1] * 1000 + buff[2] * 100 + buff[3] * 10 + buff[4]);
+            }
```
Comment (pepol) [inline programs/driver_board/main.c]: Please change this to:

``` c
motor_speed = buff[1] * 1000 + buff[2] * 100 + buff[3] * 10 + buff[4];
if (buff[0] == 0) motor_speed *= -1;
```

Also can I just ask why `0` as negative value marker? Usually `1` is used (as in, for example, two's complement for binary numbers, where sign bit is `0` for positive values and `1` for negative ones). I'm just wondering why this choice?

**Comment by pepol** [comment]
@Meli-0xFF apart from those 2 comments, it looks good.

File: programs/driver_board/main.c (hunk 1 above)
Comment (Meli-0xFF) [inline programs/driver_board/main.c]: done

File: programs/driver_board/main.c (hunk 2 above)
Comment (Meli-0xFF) [inline programs/driver_board/main.c]: done, 0 changed to 1 :)

**Comment by pepol** [comment]
@Meli-0xFF LGTM, please squash (keep just the original commit message) and feel free to merge as well.

**Pull request merged**